README.md

```diff
@@ -165,5 +250,13 @@ relationships between the two are set up in Active Record.
 
 Write your code here:
 ```ruby
-# code here
+genie_lamp = Lamp.create(wishes_remaining: 3)
+genie = Genie.create(name:"Genie",lamp: genie_lamp)
+genie_lamp.wishes_remaining = 1
+   would you need a genie_lamp.save? I assume that since it's a class method it is not needed
+new_lamp = Lamp.create(wishes_remaining: 3)
+jafar = Genie.create(name:"Jafar",lamp: new_lamp)
+jafar.lamp = nil - I set up the whole model to see if this would work it did I thought you weren't allowed to do this? of course I did not set up NOT NULL in the schema but I thought that would be inherit for a foreign key I thought we learned that it was for a primary key.
+
```

Comment:
Active Record can be configured to have optional foreign keys, what it will not let you do by default is delete an instance of a parent model, without first deleting all of that instance's associated children. Basically, you cannot have a foreign key pointing to a record that no longer exists.